cypress.config.ts

```diff
@@ -0,0 +1,43 @@
+import { defineConfig } from "cypress";
+
+export default defineConfig({
+  defaultCommandTimeout: 10000,
+  e2e: {
+    // We've imported your old cypress plugins here.
+    // You may want to clean this up later by importing these.
+    setupNodeEvents(on, config) {
+      on("task", {
+        log(args) {
+          console.log(args);
+
+          return null;
+        },
+        table(message) {
+          console.table(message);
+
+          return null;
+        },
+      });
+      return config;
+    },
```

Comment:
`/plugins/index.ts` has been removed and is now part of this config file as `setupNodeEvents`